src/components/preview/PersonalInfo.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useState } from 'react';
+
+const PersonalInfo = props => {
+  const { personalInfo, preview } = props;
+
+  const personalInfoList = Object.keys(personalInfo).map(key => (
+    <PersonalInfoItem key={key} title={key} data={personalInfo[key]} preview={preview}></PersonalInfoItem>
+  ));
+
+  return (
+    <div>
+      <div>
+        <h1>Personal Information</h1>
+        <span>Edit</span>
+      </div>
+      <ul>{personalInfoList}</ul>
+    </div>
+  );
+};
+
+PersonalInfo.propTypes = {
+  personalInfo: PropTypes.object,
+  preview: PropTypes.bool,
+};
+
+const PersonalInfoItem = props => {
```

Comment:
It would be good if you could separate this code in a new file

Reply:
git commit -m "form-preview:#2 - removed PersonalInfoItem to separate file"